docs/source/faq.rst

```diff
@@ -97,3 +97,32 @@ For instance, you can stop showing each trial result as follows:
 
 
 Please refer to :class:`optuna.logging` for further details.
+
+
+How to save a machine learning model trained in an objective function?
```

Comment:
I'm not good at English, but I personally felt the following might be more natural: `How to save machine learning models trained in objective functions?` as the objective function is always called repeatedly and thus more than one model will be saved.

Reply:
Thank you for your suggestion. That makes sense. I'll fix it.